gf-guide/metrics.md

```diff
@@ -116,12 +121,10 @@ Exceptions are usually made if the font’s primary script isn’t Latin, Greek
 
 Please keep in mind that this calculation is to be set according to the specificities of each font.
 
--   The 120% suggested above is for compatibility with DTP apps. Still, it can often be too tight if your font covers more languages than basic Latin, Greek, and Cyrillic, or if you have a particular design with short ascenders.
--   The choice of the `Agrave` is purely based on the behaviour of some applications, it is not based on some general rules of design.
--   Google Fonts is trying to push designers to include proper support of the mark-to-mark feature allowing combination of diacritics and display of non-encoded accented glyphs. Pay attention to your anchor placement so that, if you combine breve and acute for example, you don't end up with a severe interline glyph clashing. Or adapt your vertical metrics with a certain measure, to find the best compromise.
--   Google Fonts wishes to update fonts to expand glyhpsets, and therefore language support, and therefore accessibility. If your intention is to have, for example, Vietnamese coverage or Thai script in a next update, you can already anticipate the vertical metrics to avoid regressions later.
+-   The 120% suggested above is for compatibility with DTP apps. Still, it can often be too tight if your font covers more languages than basic Latin, Greek, and Cyrillic, or if you have a particular design with short ascenders.  
+-   Please pay careful attention to the overall design of diacritic marks, especially the [stacked diacritics](https://googlefonts.github.io/gf-guide/diacritics.html#stacked-diacritics). This way you ensure the ascenders value is not unnecessarily high. 
```

Comment:
We could be more precise about that technique like clearly saying that "the line spacing is a design decision and should be a compromise between good legibility and avoiding glyphs clashes in between lines. A good practice is to determine an ideal line spacing and then draw the diacritics accordingly or adapt them to it."

Reply:
Adding the first part to requirement 1. Second here in the note.